test-files/cases/attribute/unifying/highway-2888/Expected.osm

```diff
@@ -4709,6 +4709,55 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
+    <way visible="true" id="-1000" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-111"/>
+        <nd ref="-109"/>
+        <nd ref="-108"/>
+        <nd ref="-107"/>
+        <nd ref="-135"/>
+        <nd ref="-1035"/>
+        <tag k="length" v="141.8"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-1000"/>
+    </way>
+    <way visible="true" id="-984" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-1107"/>
+        <nd ref="-1106"/>
+        <tag k="length" v="54.3"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-984"/>
+    </way>
```

Comment:
Way `-984` is an issue with using a corner splitter with bad data.  The "dog leg" in this crappy way is split into its own east/west way while the actual way runs north/south.  Once it is isolated by itself it no longer is conflated to any NS ways and thus doesn't get the correct attributes.  One solution is to leave the corner splitter as-is and another would be to keep the corner splitter from splitting off way segments that contains only two nodes is shorter than the circular error.  This could potentially affect a lot of conflation because the corner splitter is default for network conflation but it would allow for ways with dog-legs at the end to conflation with straight ways.  I'm leaning towards adding that in.  What do you think?
![2888-way-984](https://user-images.githubusercontent.com/13385275/53366880-474a0400-390a-11e9-9e55-3a5831983c99.png)

Reply:
Let's try that change.  If it helps Attribute Conflation only, it could also be an option that gets enabled for Attribute Conflation and turned off for everything else.

Reply:
This change seemed to help all over the place.  There are lots of places where little "dog-legs" caused issues in conflation and now don't.